support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -392,7 +392,7 @@ export function ThreeTierLanding(): JSX.Element {
 			/**
 			 * Only Testing CardTier1 wth checkout
 			 */
-			if ((useGenericCheckout && cardTier === 1) || tier2UseGenericCheckout) {
+			if (useGenericCheckout) {
```

Comment:
[`cardTier === 3` is handled above](https://github.com/guardian/support-frontend/pull/6236/files#diff-9e9d19a3470db4ff48cd4c73865be05d061001afc0751360451d6d509f390e9bR363), thus this is only `2 || 3`. 

I would prefer if this was explicit incase we added a tier (unlikely), but eslint complains with a [`no-unnecessary-condition`](https://typescript-eslint.io/rules/no-unnecessary-condition/) error.